types/puppeteer/index.d.ts

```diff
@@ -804,6 +804,15 @@ export interface FrameBase {
   /** Adds a `<link rel="stylesheet">` tag into the page with the desired url or a `<style type="text/css">` tag with the content. */
   addStyleTag(options: StyleTagOptions): Promise<void>;
 
+  /**
+   * This method fetches an element with selector, scrolls it into view if needed, and
+   * then uses `page.mouse` to click in the center of the element. If there's no element
+   * matching selector, the method throws an error.
+   * @param selector A selector to search for element to click. If there are multiple elements satisfying the selector, the first will be clicked.
+   * @param options Specifies the click options.
+   */
+  click(selector: string, options?: ClickOptions): Promise<void>;
```

Comment:
Now that both `Frame` and `Page` have a `click` method we can finally remove the one for the `Page` class https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25508/files#diff-26c1cae70a0e01ff10cdc4e82232b3eaL952

Reply:
Done 👍